packages/x-tree-view/src/internals/TreeViewProvider/useTreeViewContext.ts

```diff
@@ -3,5 +3,17 @@ import { TreeViewAnyPluginSignature } from '../models';
 import { TreeViewContext } from './TreeViewContext';
 import { TreeViewContextValue } from './TreeViewProvider.types';
 
-export const useTreeViewContext = <TPlugins extends readonly TreeViewAnyPluginSignature[]>() =>
-  React.useContext(TreeViewContext) as TreeViewContextValue<TPlugins>;
+export const useTreeViewContext = <TPlugins extends readonly TreeViewAnyPluginSignature[]>() => {
+  const context = React.useContext(TreeViewContext) as TreeViewContextValue<TPlugins>;
+  if (context == null) {
+    throw new Error(
+      [
+        'MUI: Could not find the tree view context.',
+        'It looks like you rendered your component outside of a TreeView or RichTreeView parent component.',
```

Comment:
Shouldn't this be `SimpleTreeView`?
```suggestion
        'It looks like you rendered your component outside of a SimpleTreeView or RichTreeView parent component.',
```